index.html

```diff
@@ -674,15 +674,20 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 the last request.</p>
 </aside>
 <p data-dfn-for="PerformanceResourceTiming">On getting, the
-<dfn>responseEnd</dfn> attribute MUST return as follows:</p>
+<dfn>responseEnd</dfn> attribute MUST return the result of running
+<a>get response end time</a> algorithm.</p>
+
+<p>When requested to run the <dfn>get response end time</dfn>
+algorithm, the user agent must run the following steps:</p>
 <ol>
-<li>The time immediately after the user agent receives the last
-byte of the response or immediately before the transport connection
-is closed, whichever comes first. The resource here can be received
-either from <a data-cite="HTML#relevant-application-cache">relevant
-application caches</a>, local resources, or from the server.</li>
-<li>The time immediately before the user agent aborts the fetch due
-to a network error.</li>
+<li>If the fetch was aborted due to a network error, return the time
+immediately before the user agent aborts the fetch.</li> 
+<li>Otherwise, return the time immediately after the user agent
+receives the last byte of the response or immediately before the
```

Comment:
This can be more algorithmic:
Let timeA be ...
Let timeB be...
Return the mininum between timeA and timeB.

Reply:
I agree that it can, but not sure it's of huge value as the times in question are not well-defined, and we will need to better define them through Fetch integration.

Reply:
It has the value of improving the spec 'health' a little bit and more clarity. But I want this change to land anyways, so it's fine if we keep as is for now.

Reply:
I agree, but I expect this to change significantly (and don't think rewriting it now would add significant short-term value).